ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/less/signin.less

```diff
@@ -9,6 +9,7 @@
 @rememberms-font-color:#7c7c7c;
 
 .cmp-login {
+	font-family: Arial, Helvetica, sans-serif;
```

Comment:
@rakeshkumar-verma : we are using the font family with san-serif as priority in button.css . why have we changed priority here ?

Reply:
Updated back to san-serif

Reply:
@rishabh-behl  updated to  sans-serif;